modules/createHashHistory.js

```diff
@@ -33,7 +37,7 @@ var DefaultQueryKey = '_k';
 function createHashHistory(options={}) {
   var { queryKey } = options;
 
-  if (queryKey && typeof queryKey !== 'string')
+  if (typeof queryKey !== 'string')
```

Comment:
This is bug, because if I don't pass anything then queryKey is undefined and DefaultQueryKey is no set.